src/OpenTelemetry/Metrics/ExponentialHistogramBuckets.cs

```diff
@@ -43,6 +43,15 @@ internal void SnapshotBuckets(CircularBufferBuckets buckets)
         buckets.Copy(this.buckets);
     }
 
+    internal ExponentialHistogramBuckets Copy()
+    {
+        var copy = new ExponentialHistogramBuckets();
+        copy.Offset = this.Offset;
+        copy.buckets = new long[this.buckets.Length];
+        Array.Copy(this.buckets, copy.buckets, this.buckets.Length);
```

Comment:
Copy `size`?

Reply:
Good catch, will fix.

Reply:
Turns out this was actually more insidious... I removed `size` altogether. Size was initialized wrong... this was never testing anything correctly.

This was a result of a refactor in my prior PR where I changed `ExponentialHistogramBuckets` from having a handle an instance of `CircularBufferBuckets` to simply a `long[]`.